### DISCUSSION TITLE
Multiple local folder to one virtual folder?

### DISCUSSION THREAD

**Discussion by Markovski2338** [rejetto/hfs#686] [Q&A]
Is there a way to point multiple local folder to one virtual folder?

**Comment by rejetto** [comment]
this is not supported, but i may work on it if many people need it.
in the meantime, you can consider an alternative solution: adding those folders as subfolders to one virtual folder, then use the "search" button to access their content as a single list.
if you don't like clicking on the search button, you can make a web-link to the folder adding `?search=*` in the address. 

<img width="546" alt="image" src="https://github.com/user-attachments/assets/30c89637-8036-404e-a42a-6a3901cc1200">

**Reply by Markovski2338** [reply]
Thank you! 
The work around works but it is very messy as it outputs all folders then all sub folders and then all files(as it should since we are searching for "all", it works just not ideal for organizing). It would very nice to implement this feature into HFS as it would make organizing the file structure a whole lot easier!

**Comment by rejetto** [comment]
if you don't like folders being included in the result, i can easily introduce a feature for that (i already have an option onlyFolders, but not onlyFiles).
in the meantime, you can consider searching not * but . as `?search=.`
as almost any file will have the dot, and most folders don't, so it may even work in your case.

**Reply by Markovski2338** [reply]
okay cool! Is there a way to have 2 parameters to search for? (ex. ?search=.mkv , .mp4) and is there a way to exclude a char from the search? (ex. ?search=* & != "/")

**Comment by rejetto** [comment]
Yes but use pipe | as delimiter instead of comma ,

**Comment by rejetto** [comment]
https://github.com/rejetto/hfs/wiki/Wildcards

**Reply by Markovski2338** [reply]
Thank you!
